website/src/components/parts/FirstSection.js

```diff
@@ -1,74 +0,0 @@
-import React from "react";
```

Comment:
@ntolia did you actually mean to delete these?

Reply:
Not seeing the corresponding properly named component files. Maybe a different PR?

Reply:
Nenermind. They are already there.